vision/product_search/product_search_get_reference_image.rb

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018 Google, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START vision_product_search_get_reference_image]
+require "google/cloud/vision"
+
+def product_search_get_reference_image(
+  project_id = "your-project-id",
+  location   = "us-west1",
+  product_id = "your-product-id",
+  image_id   = "your-image-id"
+)
+  client               = Google::Cloud::Vision::ProductSearch.new
+  reference_image_path = client.class.reference_image_path project_id, location, product_id, image_id
```

Comment:
@beccasaurus this line is fairly long. Should I make sure lines are under 80 characters like python? What's your preferred way of breaking this into multiple lines?

Reply:
Spoke in person about this and updated in 3fda682